vmdk_plugin/Makefile

```diff
@@ -56,6 +57,7 @@ AFTER_REMOVE   := $(SCRIPTS)/install/after-remove.sh
 #
 CHECK	    := $(SCRIPTS)/check.sh
 BUILD       := $(SCRIPTS)/build.sh
+BUILD_TOOLS  := $(SCRIPTS)/deploy-tools.sh
```

Comment:
the script is calleed deploy_tools. Why the Makefile vars is called build_tools ? it's confusing